src/Shared/LayoutRenderers/AspNetResponseCookieLayoutRenderer.cs

```diff
@@ -93,6 +105,92 @@ private HttpCookieCollection GetCookies(HttpResponseBase response)
             var expandMultiValue = OutputFormat != AspNetRequestLayoutOutputFormat.Flat;
             return HttpCookieCollectionValues.GetCookieValues(cookies, CookieNames, Exclude, expandMultiValue);
         }
+
+        private IEnumerable<HttpCookie> GetVerboseCookieValues(HttpCookieCollection cookies)
+        {
+            var expandMultiValue = OutputFormat != AspNetRequestLayoutOutputFormat.Flat;
+            return HttpCookieCollectionValues.GetVerboseCookieValues(cookies, CookieNames, Exclude, expandMultiValue);
+        }
+
+        private void SerializeAllProperties(IEnumerable<HttpCookie> verboseCookieValues, StringBuilder builder, LogEventInfo logEvent)
+        {
+            switch (OutputFormat)
+            {
+                case AspNetRequestLayoutOutputFormat.Flat:
+                    SerializeAllPropertiesFlat(verboseCookieValues, builder, logEvent);
+                    break;
+                case AspNetRequestLayoutOutputFormat.JsonArray:
+                    SerializeAllPropertiesJson(verboseCookieValues, builder);
+                    break;
+            }
+        }
+
+        private void SerializeAllPropertiesJson(IEnumerable<HttpCookie> verboseCookieValues, StringBuilder builder)
+        {
+            var firstItem = true;
+
+            foreach (var cookie in verboseCookieValues)
+            {
+                if (firstItem)
+                {
+                    builder.Append('[');
+                }
+                else
+                {
+                    builder.Append(',');
+                }
+
+                builder.Append("{");
+
+                AppendJsonProperty(builder, "Name", cookie.Name);
```

Comment:
Can you change `AppendJsonProperty` to return the input-builder, so you can do this:
```c#
AppendJsonProperty(builder, nameof(cookie.Name), cookie.Name).Append(',');
```

Reply:
changed to return StringBuilder for future use